### PR TITLE
SUBMARINE-649. Disable incomplete pages in 0.5.0

### PR DESCRIPTION
**Pull request by ByronHsu**
### What is this PR for?

1. Redirect root page to experiment/
2. Disable route and button for data/, model/, workspace/, interpreter/

### What type of PR is it?

[Hot Fix]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-649

### How should this be tested?
https://travis-ci.org/github/ByronHsu/submarine/builds/731919190

### Screenshots (if appropriate)
Screen Shot 2020-10-01 at 8 35 19 PM

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Review by xunliu — COMMENTED** [review]
@ByronHsu Please added `SUBMARINE-649` to PR title.
like., SUBMARINE-649. Disable incomplete pages in 0.5.0

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.ts [hunk 1]
```diff
@@ -40,19 +41,23 @@ interface SidebarMenu {
...
+
 export class WorkbenchComponent implements OnInit {
   isCollapsed: boolean = false;
   highlighted: boolean = true;
   menus: SidebarMenu[] = [
     {
       title: 'Home',
       iconType: 'home',
-      routerLink: '/workbench/home'
+      routerLink: '/workbench/home',
```
Comment (jiwq) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.ts:54]:
```suggestion
      routerLink: '/workbench/home'
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench-routing.module.ts [hunk 2]
```diff
@@ -29,6 +29,13 @@ import { InterpreterComponent } from './interpreter/interpreter.component';
 import { ModelComponent } from './model/model.component';
 import { WorkspaceComponent } from './workspace/workspace.component';
 
+function disablePage(routes: Routes): Routes{
```
Comment (jiwq) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench-routing.module.ts:32]:
```suggestion
function disablePage(routes: Routes): Routes {
```

**Review by jiwq — CHANGES_REQUESTED** [review]
Thanks @ByronHsu for the work. Please check my request changes.

**Comment by ByronHsu** [comment]
@jiwq Thanks for your help! 
I'm using vscode to develop, and also install the prettier package.
However, it did not detect this formatting problem automatically.
Is this normal? Or maybe my vscode setting is wrong.

**Comment by kevin85421** [comment]
@ByronHsu Thank you for your contribution! 

The home page is not included in release 0.5, and thus needs to be disabled too.

**Comment by ByronHsu** [comment]
@kevin85421 ok! I have added it!

**Comment by kevin85421** [comment]
This issue is blocked by #425, and thus #425 should be merged to master prior to this issue. With the rebase of #425, this issue should be able to pass the E2E integration tests.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html [hunk 3]
```diff
@@ -47,8 +47,9 @@ <h1>Submarine</h1>
         [nzDisabled]="menu.disabled"
         [nzTitle]="menu.title"
         nzPlacement="right"
-        [nzMatchRouter]="!menu.disabled"
-        [routerLink]="menu.disabled ? null : menu.routerLink"
+        nzMatchRouter
+        [routerLink]="menu.routerLink"
+        [nzDisabled]="menu.disabled"
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workbench.component.html:52]:
```suggestion
```
Duplicate with line 47

**Review by pingsutw — COMMENTED** [review]
Thanks @ByronHsu for the contribution

**Review by pingsutw — APPROVED** [review]
LGTM

**Pull request closed without merging**